pkg/controllers/syncer/testing/manager.go

```diff
@@ -30,7 +30,7 @@ type fakeManager struct {
 
 func (f *fakeManager) SetFields(interface{}) error { return nil }
 
-func (f *fakeManager) GetConfig() *rest.Config { return &rest.Config{Host: "127.0.0.1"} }
+func (f *fakeManager) GetConfig() *rest.Config { return &rest.Config{Host: "unit-test-client"} }
```

Comment:
is it still used?

Reply:
I think this actually is still a good idea as this shows up in tests a lot nicer than 127.0.0.1